drivers/gpu/drm/xe/xe_pci.c

```diff
@@ -336,7 +336,6 @@ static const struct xe_device_desc mtl_desc = {
 static const struct xe_device_desc lnl_desc = {
 	PLATFORM(XE_LUNARLAKE),
 	.has_display = true,
-	.require_force_probe = true,
```

Comment:
Don't we have similar patch in linux-next to sherry-pick?

Reply:
Let me check, when I submitted this (one month ago), there wasn't anything in linx-next yet. Btw, sherry-picking sounds fun :)

Reply:
@ujfalusi nope, not yet https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/tree/drivers/gpu/drm/xe/xe_pci.c#n338   ... still requires force_probe as of linux-next 20240816